speak/face.py

```diff
@@ -152,20 +144,37 @@ def update(self, status=None):
             self._eyebox.pack_start(the, True, True, FACE_PAD)
             the.show()
 
-        self._mouth = status.mouth(self._audio, self.fill_color)
+        self._mouth = status.mouth(self.fill_color)
         self._mouth.show()
         self._mouthbox.add(self._mouth)
 
     def say(self, something):
-        self._audio.speak(self._peding or self.status, something)
+        if self._pending is None:
+            voice_name = self.status.voice.name
+        else:
+            voice_name = self._pending.voice.name
+        all_voices = self.speech.get_all_voices()
```

Comment:
Several lines of `say` look just like several lines of `say_notification`.

Reply:
I have resolved that by creating a separate function thats called from both `say` and `say_notification`.